dev-python/pgspecial/pgspecial-1.12.1.ebuild

```diff
@@ -3,7 +3,7 @@
 
```

Comment:
Could you please split pgspecial into a separate PR and attempt to start a server for tests (similarly to how psycopg does it)? Almost all tests are skipped right now.

Reply:
sure